src/main/groovy/com/bmuschko/gradle/java2html/ConvertCodeTask.groovy

```diff
@@ -30,23 +30,23 @@ import org.gradle.api.tasks.TaskAction
  */
 class ConvertCodeTask extends DefaultTask {
     @InputFiles FileCollection classpath
-    @Input FileCollection srcDirs
+    @InputFiles FileCollection srcDirs
     @OutputDirectory File destDir
     @Input String includes
-    String outputFormat
-    Integer tabs
-    String style
-    Boolean showLineNumbers
-    Boolean showFileName
-    Boolean showDefaultTitle
-    Boolean showTableBorder
-    Boolean includeDocumentHeader
-    Boolean includeDocumentFooter
-    Boolean addLineAnchors
-    String lineAnchorPrefix
-    String horizontalAlignment
-    Boolean useShortFileName
-    Boolean overwrite
+    @Input String outputFormat
```

Comment:
I'd think that at least if not all of them are optional and should use `@Optional`?

Reply:
That's what I thought too but then saw all properties have default values provided by the plugin convention, even those that were previously annotated with `@Input`.